beetmoverscript/constants.py

```diff
@@ -155,6 +158,11 @@
     'thunderbird': 'pub/thunderbird/',
 }
 
-PARTNER_LEADING_STRING = 'releng/partner/'
-PARTNER_REPACK_PUBLIC_PREFIX_TMPL = 'pub/firefox/candidates/{version}-candidates/build{build_number}/partner-repacks/'
-PARTNER_REPACK_PUBLIC_PAYLOAD_ID = 'is_partner_repack_public'
+PARTNER_REPACK_PUBLIC_PREFIX_TMPL = 'pub/firefox/candidates/{version}-candidates/build{build_number}/'
+PARTNER_REPACK_PRIVATE_REGEXES = (
+    r'^[^\/.]+/{version}-{build_number}/[^\/.]+\/(mac|win32|win64|linux-i686|linux-x86_64)/[^\/.]+$',
+)
+PARTNER_REPACK_PUBLIC_REGEXES = (
+    r'^(mac|win32|win64)-EME-free\/[^\/.]+$',
```

Comment:
Ah, had this on my TODO list and forgot to mention it. I assumed we could trim that `v1` from the first iteration. This is much better though 👍